setup.py

```diff
@@ -29,7 +30,8 @@
         "Development Status :: 4 - Beta",
     ],
     install_requires=[
-        'Twisted >=10.0.0, <=12.3',
+        'cryptography>=2.1.4',
```

Comment:
We were clamping twisted to <=12.3, now we are not clamping the upper bound, so we'll be jumping 6 years of twisted release. Any idea on how we will be able to tell if this works without just running it?

Reply:
well we're able to pass integration tests on py2.7, which builds a deb, installs it, runs trond and sends some api requests - all of it works. for sure we will need to do a more thorough testing on playground / dev / stage.